Components/Image/image.css

```diff
@@ -0,0 +1,29 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
```

Comment:
keep resets in separate file

Reply:
Actually, i am allowing users to import CSS file for only 1 component too, that's why i kept it there. I can use import too but it was just 2 lines so i didn't.

Reply:
Okay!